docs/installation.rst

```diff
@@ -18,6 +18,12 @@ With pip installed, you can do the following:
 Getting Started
 ===============
 
+.. note::
+   FuncX is a major component of Glaider, using FuncX Endpoints to execute
+   functions defined in Gladier Tools. See
+   `First Time Setup <https://funcx.readthedocs.io/en/latest/endpoints.html#first-time-setup>`_
+   for how to setup a local endpoint you can use for running these examples.
+
```

Comment:
We also have the funcx example on the gladier-client-template. Adding here for future reference.

Reply:
I think that would be good for people interested in authoring new Gladier Tools. For Getting Started, I think it's good to minimize the amount of new information we give users to make running examples as easy as possible.